packages/cli/test/integration/shared/utils/index.ts

```diff
@@ -86,6 +87,10 @@ export async function initNodeTypes() {
 			type: new ScheduleTrigger(),
 			sourcePath: '',
 		},
+		'n8n-nodes-base.formTrigger': {
+			type: new FormTrigger(),
+			sourcePath: '',
+		},
```

Comment:
We keep this as lean as possible to keep the tests fast. I guess having one webhook based trigger in here makes sense, but maybe it should be the webhook trigger and not the form trigger. 

For the test it doesn't make a difference because all the webhook stuff is mocked.

Reply:
Refactored to pass in nodes if required to avoid a performance impact with all integration tests that use the `initNodeTypes` helper.